src/Microsoft.AspNet.BasicMiddleware/OverrideHeaderMiddlewareOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.BasicMiddleware
+{
+    public class OverrideHeaderMiddlewareOptions
+    {
+        public OverrideHeaderMiddlewareOptions()
+        {
+            XForwardedForEnabled = true;
```

Comment:
You can call a constructor from another one. In you case, you can just call the other constructor:

```
public OverrideHeaderMiddlewareOptions()
    :this(allOverridesEnabled: true) {
}
```